docs/_static/js/auto_module_index.js

```diff
@@ -10,15 +10,17 @@ function auto_index(module) {
     var html = "<ul>";
 
     for (var i = 0; i < targets.length; ++i) {
-      var id = $(targets[i]).attr('id');
-      // remove 'mxnet.' prefix to make menus shorter
-      var id_simple = id.replace(/^mxnet\./, '');
-      html += "<li><a class='reference internal' href='#";
-      html += id;
-      html += "'>" + id_simple + "</a></li>";
+	var id = $(targets[i]).attr('id');
+	if ( id ) {
```

Comment:
So what was the issue here? This check makes the difference between the page rendering or not?

Reply:
I added a short note in the PR section describing the change. Please feel free to correct my understanding if I am wrong :)

Reply:
You are correct,
http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-13246/1/api/python/ndarray/ndarray.html

page wouldn't render in the previous PR but would render here.